### ISSUE TITLE
Make release should build statically

### ISSUE THREAD

**Issue by neuroscr** [loki-project/loki-network#823]
Jeff approved said make an issue

**Comment by despair86** [comment]
only dynamic dependencies should be libc and whatever system libs libuv pulls in (if at all)
on all other platforms, provided a static libuv is available, this is already true.
```
sun-pc% ldd lokinet
        libumem.so.1 =>  /lib/64/libumem.so.1
        libkstat.so.1 =>         /lib/64/libkstat.so.1
        libsendfile.so.1 =>      /lib/64/libsendfile.so.1
        libsocket.so.1 =>        /lib/64/libsocket.so.1
        libnsl.so.1 =>   /lib/64/libnsl.so.1
        libm.so.2 =>     /lib/64/libm.so.2
        librt.so.1 =>    /lib/64/librt.so.1
        libc.so.1 =>     /lib/64/libc.so.1
        libmp.so.2 =>    /lib/64/libmp.so.2
        libmd.so.1 =>    /lib/64/libmd.so.1
```

**Comment by despair86** [comment]
on windows:
```
[despair@NT-DEV build]$ ldd lokinet
        ntdll.dll => /c/WINDOWS/SYSTEM32/ntdll.dll (0x771e0000)
        KERNEL32.DLL => /c/WINDOWS/System32/KERNEL32.DLL (0x76810000)
        KERNELBASE.dll => /c/WINDOWS/System32/KERNELBASE.dll (0x76b10000)
        ADVAPI32.dll => /c/WINDOWS/System32/ADVAPI32.dll (0x76410000)
        msvcrt.dll => /c/WINDOWS/System32/msvcrt.dll (0x76a50000)
        sechost.dll => /c/WINDOWS/System32/sechost.dll (0x762b0000)
        RPCRT4.dll => /c/WINDOWS/System32/RPCRT4.dll (0x76690000)
        SHLWAPI.dll => /c/WINDOWS/System32/SHLWAPI.dll (0x763b0000)
        IPHLPAPI.DLL => /c/WINDOWS/SYSTEM32/IPHLPAPI.DLL (0x748c0000)
        USER32.dll => /c/WINDOWS/System32/USER32.dll (0x76490000)
        win32u.dll => /c/WINDOWS/System32/win32u.dll (0x75d70000)
        GDI32.dll => /c/WINDOWS/System32/GDI32.dll (0x75e10000)
        gdi32full.dll => /c/WINDOWS/System32/gdi32full.dll (0x75700000)
        msvcp_win.dll => /c/WINDOWS/System32/msvcp_win.dll (0x76190000)
        ucrtbase.dll => /c/WINDOWS/System32/ucrtbase.dll (0x76070000)
        WS2_32.dll => /c/WINDOWS/System32/WS2_32.dll (0x769e0000)
```

**Comment by despair86** [comment]
on linux, a dynamic libuv is fine because libuv itself is managed by apt
```
        linux-gate.so.1 (0xb7fd1000)
        libuv.so.1 => /lib/i386-linux-gnu/libuv.so.1 (0xb7aa3000)
        libstdc++.so.6 => /lib/i386-linux-gnu/libstdc++.so.6 (0xb78c6000)
        libm.so.6 => /lib/i386-linux-gnu/libm.so.6 (0xb77c2000)
        libgcc_s.so.1 => /lib/i386-linux-gnu/libgcc_s.so.1 (0xb77a4000)
        libpthread.so.0 => /lib/i386-linux-gnu/libpthread.so.0 (0xb7784000)
        libc.so.6 => /lib/i386-linux-gnu/libc.so.6 (0xb75a4000)
        librt.so.1 => /lib/i386-linux-gnu/librt.so.1 (0xb759a000)
        libdl.so.2 => /lib/i386-linux-gnu/libdl.so.2 (0xb7594000)
        /lib/ld-linux.so.2 (0xb7fd2000)
```

**Comment by despair86** [comment]
fixed in #829
